navigation_and_routing/pubspec.yaml

```diff
@@ -0,0 +1,76 @@
+name: navigation_and_routing
```

Comment:
nit: Remove autogenerated comments. Would be good to change description as well.